src/locale.en.js

```diff
@@ -78,6 +80,25 @@ export const reducerAlerts = {
     site_upload_error: 'Sorry, there was an error uploading your site. Please try again later.'
 };
 
+export const search_screen = {
+    title: 'Search',
+    header_title: 'Find a Site',
+    filter: 'Filter'
+};
+
+export const site_detail_screen = {
+    header_title: 'Site Detail'
+};
+
+export const filter_screen = {
+    results: 'Results'
+};
+
+export const add_site_screen = {
+    title: 'Add a Site',
+    header_title: 'Your Submitted Sites'
+};
+
```

Comment:
Several places that I had left strings. I now created language values and am using these instead.